.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,51 @@
+<!--- Title your PR with a Jira reference (if available) followed by brief description - for example: "PAYARA-1234 Add readme file" -->
+
+## Description
+This is a bug fix / feature. <!-- delete/modify as applicable-->
+
+<!-- Provide some context here -->
+
+<!--- Please provide enough information here about the what and why of your change. Target for developers of any experience level to understand -->
+
+## Important Info
+
+### Dependant PRs (if applicable)
+<!--- Link any related or dependant PRs here with brief description why -->
+
+### Blockers (if applicable)
+<!-- Detail any blockers with links/info -->
+
+### Deadline (if applicable)
+<!-- Detail any deadlines, why? -->
+
+## Testing
+
+### New tests
+<!-- Link to the test suite PR or provide info -->
+
+### Testing Performed
+<!--- Please describe how you tested these changes.  -->
+
+### Test suites executed
+<!-- Which test suites did you run this against? put an 'x' in the appropriate box(s). Feel free to add others.-->
+- [ ] Quicklook
+- [ ] Payara Samples
+- [ ] Java EE7 Samples
+- [ ] Java EE8 Samples
+- [ ] Payara Private Tests
+- [ ] Payara Microprofile TCKs Runner
+- [ ] Jakarta TCKs
+- [ ] Microprofile Samples 1.2
+- [ ] Microprofile Samples 1.4
+- [ ] Mojarra
+- [ ] Cargo Tracker
+
+### Testing Environment
+<!--- Which OS, JDK, Maven version did you use? - for example "Zulu JDK 1.8_212 on Ubuntu 18.04 with Maven 3.6.0"-->
+
+## Documentation
+<!-- Link to the documentation PR -->
```

Comment:
Link to the docs repo please - you're assuming knowledge and not all PRs are made by us.

Reply:
Side note - should this just be a subheading under dependant PRs?

Reply:
Adding "where applicable". 

Dependant PRs is too generic. Idea of this section is to put specific thought to documentation, and have a section you can instantly find documentation change link